catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/GetRelatedFilesRequestImpl.java

```diff
@@ -131,23 +131,23 @@ private void evaluateThumbnailNames(List<String> fileNames, Metacard metacard) {
   }
 
   @Override
-  public RequestDescription get_request_description() throws ProcessingFault, SystemFault {
```

Comment:
❗️ You can't rename these methods. They override interface methods in generated CORBA api code.